backend/app/services/users/sign_in.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Users
+  class SignIn < Core::Service
+    def call(email:, password:)
+      @email = email
+      @password = password
+      @user = yield find_user(@email)
```

Comment:
No need to make `user` an instance variable. Just make it local. You're using this variable only for `issue_token` and you are providing it there via argument.

Reply:
Also, you could skip passing email there as it available via reader